src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor();
+        Random random = new Random();
+        int number = random.nextInt(101);
```

Comment:
Magic number 101 should be replaced with a named constant to improve code readability and maintainability.

Reply:
magic number